api/pom.xml

```diff
@@ -36,11 +36,11 @@
         <legal.doc.source>../</legal.doc.source>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
+        <maven.compiler.source>11</maven.compiler.source>
```

Comment:
Cannot move up since 4.0 is part of Jakarta EE 9 which requires Java 8 support.

Reply:
@tkburroughs Agree.  Adding a standard module-info or changing the JVM version would require a 4.1.  I asked Scott on https://github.com/eclipse-ee4j/jakartaee-platform/issues/421 where the appropriate place is to have the discussion on adding module-info.java files to all the API jars.

Reply:
I have changed this back to 1.8 and added multi-release jar support. The module-info will be visible only JDK11+.